rules/swss.mk

```diff
@@ -4,7 +4,7 @@ SWSS = swss_1.0.0_$(CONFIGURED_ARCH).deb
 $(SWSS)_SRC_PATH = $(SRC_PATH)/sonic-swss
 $(SWSS)_DEPENDS += $(LIBSAIREDIS_DEV) $(LIBSAIMETADATA_DEV) $(LIBTEAM_DEV) \
     $(LIBTEAMDCT) $(LIBTEAM_UTILS) $(LIBSWSSCOMMON_DEV)
-$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM) $(LIBSWSSCOMMON) $(PYTHON_SWSSCOMMON)
+$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM) $(LIBSWSSCOMMON) $(PYTHON_SWSSCOMMON) $(LIBTEAMDCT)
```

Comment:
Notes for a future change (I don't want to hold this PR up, as it fixes the PR check builds. Making this change now would cause a multi-hour delay while the check builds run again):

I think the variable name should be changed to `LIBTEAMDCTL` (add an 'L' at the end), and the order here should be changed to:

```
$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM) $(LIBTEAMDCTL) $(LIBSWSSCOMMON) $(PYTHON_SWSSCOMMON) 
```

Reply:
Fixed.